src/sentry/integrations/vsts/issues.py

```diff
@@ -131,3 +131,27 @@ def sync_assignee_outbound(self, external_issue, user, assign=True, **kwargs):
                     'issue_key': external_issue.key,
                 }
             )
+
+    def sync_status_outbound(self, external_issue, is_resolved, project_id, **kwargs):
+        project_integration = ProjectIntegration.objects.get(
+            integration_id=external_issue.integration_id,
+            project_id=project_id,
+        )
+
+        status_name = 'resolve_status' if is_resolved else 'regression_status'
+        try:
+            status = project_integration.config[status_name]
+        except KeyError:
+            return  # TODO(lb): what's the expected behavior here?
```

Comment:
I imagine do nothing, if it's not configured. Maybe we can later add some kind of error reporting later, but this seems fine